### DISCUSSION TITLE
Getting 403 error - unable to pass the user credentials to Chat Engine API

### DISCUSSION THREAD

**Discussion by akhil9281** [axios/axios#4601] [Q&A]
I am getting " your credentials are incorrect" error on my chat dashboard.  I have tried adding / to the path , checked all my keys, replacing displayName with user. email in the get request. But the error still persists. Also, I have a lot of 403 errors in the console.

https://rest.chatengine.io/ - link of the Chat Engine API
https://youtu.be/Bv9Js3QLOLY - link for the origin source of the project

There is something not working in my **axios.get** method, I just don't know what

![2dc0c115-3876-430b-9400-bc5e95c4425a](https://user-images.githubusercontent.com/88153463/163319863-8ad05d06-c2d5-44af-bfc7-11eb3f569405.jpg)

![859349b3-10bd-4f7d-b143-dbe208a61d42](https://user-images.githubusercontent.com/88153463/163319882-03cc467f-36a2-46c9-a7d0-b82d645f1a5c.jpg)

**Comment by jasonsaayman** [comment]
This cant be an axios issue since axios get works fine. Open an issue and we can have a look.
